src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -88,6 +89,16 @@ namespace
     }
 
     const sf::Uint64 mainChunkSize = 12;
+
+    enum WaveFormat
```

Comment:
I think you should use const integers, to remain consistent with what's already done. The enum name is not even used, all we want is to name those magical numbers.